src/App.js

```diff
@@ -10,30 +10,37 @@ class App extends Component {
         super();
 
         this.state = {
-            data: null
+            inputData: [],
+            inputValue: ''
         }
     }
 
-    getInputData = (value) => {
-        this.setState({data: value});
-        console.log(value);
+    handleSubmit = (value) => {
+        let { inputData } = this.state;
+        if(value && value.length > 0){
```

Comment:
You can do this check before calling handleSubmit callback. And I think that handleSubmit - is an example of a bad naming. Because in this component you do not have any forms. It's better to call this method "handleTaskAdd" or something like this.